### DISCUSSION TITLE
求助playcover如何设置时停下干员

### DISCUSSION THREAD

**Discussion by RArturia** [MaaAssistantArknights/MaaAssistantArknights#9115] [Q&A]
用的mac模拟器playcover，按键映射也设置好了，但是选中干员拖出的时候按下暂停没有反应，好像也没有看到脚本在哪里写，救救孩子

**Comment by ABA2396** [comment]
mac没有按键输入 用不了

**Comment by RadioNoiseE** [comment]
其实是可以的。大致效果[如此](https://www.bilibili.com/video/BV12u8jzjEYN/?vd_source=c764bbae45b7ca12bcf2e208ef97418c)、当然PlayCover目前不支持宏、所以暂时做不到[这种程度](https://www.bilibili.com/video/BV1zfYseLEMq/)。
